? maemo_ignore_doxygen_tags.patch
? examples/iochannel_stream/Makefile
? examples/iochannel_stream/Makefile.in
? examples/iochannel_stream/example
? examples/options/example
? glib/glibmm/optioncontext.cc
? glib/glibmm/optioncontext.h
? glib/glibmm/optionentry.cc
? glib/glibmm/optionentry.h
? glib/glibmm/optiongroup.cc
? glib/glibmm/optiongroup.h
? glib/glibmm/private/optioncontext_p.h
? glib/glibmm/private/optionentry_p.h
? glib/glibmm/private/optiongroup_p.h
Index: docs/reference/Doxyfile.in
===================================================================
RCS file: /cvs/gnome/glibmm/docs/reference/Doxyfile.in,v
retrieving revision 1.5
diff -u -p -r1.5 Doxyfile.in
--- docs/reference/Doxyfile.in	13 Feb 2005 14:30:18 -0000	1.5
+++ docs/reference/Doxyfile.in	31 Mar 2006 14:09:01 -0000
@@ -914,7 +914,7 @@ SKIP_FUNCTION_MACROS   = YES
 # If a tag file is not located in the directory in which doxygen 
 # is run, you must also specify the path to the tagfile here.
 
-TAGFILES = "libstdc++.tag.xml = http://gcc.gnu.org/onlinedocs/libstdc++/latest-doxygen"
+#TAGFILES = "libstdc++.tag.xml = http://gcc.gnu.org/onlinedocs/libstdc++/latest-doxygen"
 
 # When a file name is specified after GENERATE_TAGFILE, doxygen will create 
 # a tag file that is based on the input files it reads.
